Server/sprawl_calculator/calculator/views.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import render
+from rest_framework.decorators import api_view
+# Create your views here.
+
+#Endpoint1_View
+@api_view(['PUT', 'GET'])
+def endpoint1(request):
```

Comment:
Same comment as above, lets rename `endpoint1` to a more readable name like e.g. `index _generator`